src/test/java/org/apache/sling/extensions/mdc/integration/ServerConfiguration.java

```diff
@@ -1,129 +0,0 @@
-/*
```

Comment:
Can you briefly explain why you have removed this class? I'd like to add it to the commit message when I merge the PR.

Reply:
@rombert I just added [description](https://github.com/apache/sling-org-apache-sling-extensions-slf4j-mdc/pull/1#issue-1226664932) 

`ServerConfiguration.java` was removed since its manual options to setup pax exam IT and its no longer required (because `org.apache.sling.testing.paxexam` comes with default options)